linera-service/src/server.rs

```diff
@@ -128,7 +126,7 @@ impl ServerContext {
                 if let Some(port) = shard.metrics_port {
                     Self::start_metrics(listen_address, &port);
                 }
-                let spawned_server = match GrpcServer::spawn(
+                let spawned_server = match grpc::Server::spawn(
```

Comment:
Here and other places, we probably want to keep full container names like `(grpc::)GrpcServer`.
This codebase has been avoiding the alternative convention`grpc::Server` quite consistently so far:
* For those not using a fancy IDE, it's a bit easier to search and replace `GrpcServer`.
* `grpc::Server` doesn't work well with `serde_reflection` (which has no namespaces).

Reply:
I wonder if it wouldn't make more sense to address these points in other ways:
- outside the current module, make a habit of referring to the server as `grpc::Server` (or whatever level of qualification is necessary to disambiguate) — this means that people without Rust IDE support can still grep for `grpc::Server` effectively 
- for `serde_reflection` we can manually collapse the namespaces back into the type with `#[serde(rename = "GrpcServer")]`, but that doesn't mean we have to also squash them in Rust where we have adequate namespacing functionality available

In the course of making this change I noticed it quite drastically cleans up a lot of imports, since

```rust
use linera_rpc::grpc::{GrpcServer, GrpcClient, GrpcError, …} 
```

becomes just

```rust
use linera_rpc::grpc;
```